ArduPlane/servos.cpp

```diff
@@ -595,6 +604,28 @@ void Plane::servo_output_mixers(void)
     }
 }
 
+/*
+  support for twin-engine planes
+ */
+void Plane::servos_twin_engine_mix(void)
+{
+    float throttle = SRV_Channels::get_output_scaled(SRV_Channel::k_throttle);
+    float rudder = SRV_Channels::get_output_scaled(SRV_Channel::k_rudder) / float(SERVO_MAX);
+    float throttle_left, throttle_right;
+    
+    if (throttle < 0 && aparm.throttle_min < 0) {
+        // doing reverse thrust
+        throttle_left  = constrain_float(throttle + 50 * rudder, -100, 0);
```

Comment:
I'm not sure how this twin engine mixer became part of this PR, but the constraint should be between the other constraints: min/mac throttle - not 0/100. Same with reverse thrust. There's some logic somewhere to handle the reverse constraint.